### PR TITLE
Add dblock class

### PR DESCRIPTION
**Pull request by wawax**
Move code related to db locks during inventory in a class
The PR https://github.com/fusioninventory/fusioninventory-for-glpi/pull/2424 needs to be merged first

### CODE REVIEW

File: inc/inventorycomputerinventory.class.php [hunk 1]
```diff
@@ -90,39 +90,14 @@ function import($p_DEVICEID, $a_CONTENT, $arrayinventory) {
...
-         <ERROR>ERROR: Timeout for DB lock based on name</ERROR>
-         </REPLY>");
-            $communication->sendMessage($_SESSION['plugin_fusioninventory_compressmode']);
-            exit;
-         }
+      if (!$dbLock->setLock('inventorynames', $name, true)) {
+         exit();
       }
-      $CFG_GLPI["use_log_in_files"] = TRUE;
+//      $CFG_GLPI["use_log_in_files"] = TRUE;
```
Comment (trasher) [inline inc/inventorycomputerinventory.class.php]: remove?

**Comment by wawax** [comment]
As there's no hurry and it might break things, I prefere to rebase it on master and not to include it on glpi9.2

**Comment by wawax** [comment]
Hi @ddurieux 
I'd like to to review this PR :)

**Comment by wawax** [comment]
I've switched the PR to master

**Pull request merged**